index.js

```diff
@@ -126,6 +126,7 @@ export default ViewMoreText = React.createClass({
     return (
       <View onLayout={this.onLayout} style={{opacity: this.state.opacity}}>
         <Text
+          style={this.props.style}
```

Comment:
Hi, this is unnecessary because you can add style to the text wrapped inside <ViewMoreText></ViewMoreText>, e.g: 

```
  <ViewMoreText>
    <Text style={/*Insert style here*/}>Lorem ispum</Text>
  </ViewMoreText>
```